uwblueprint/project-read-backend
# Create endpoints for export CSVs

### Notion ticket link



[Data export](https://www.notion.so/uwblueprintexecs/Data-export-7d5d9331d70d4bfaa25e5187617ec4d4)



### Implementation description

- Created views for export scripts



### Steps to test

1. Endpoints "export/families", "export/students", "export/fields", "export/sessions" should return CSVs of each model
